index.html

```diff
@@ -4,7 +4,8 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
-  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+<!--   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"> -->
+  <link type="text/css" rel="stylesheet" href="stylesheet.css"/>
```

Comment:
Great, we'll move in this direction as our projects become more complicated. Note that you should probably also grab that stray style declaration below (the one from the original that centers the text) and move it here as well so that you don't have custom styles in both places.